frontend/src/components/Layout.tsx

```diff
@@ -1,15 +1,9 @@
-import pick from "lodash/pick";
-
-import {
-  NextIntlClientProvider,
-  useMessages,
-  useTranslations,
-} from "next-intl";
+import { useTranslations } from "next-intl";
 import { setRequestLocale } from "next-intl/server";
 
-import Footer from "./Footer";
-import GrantsIdentifier from "./GrantsIdentifier";
-import Header from "./Header";
+import Footer from "src/components/Footer";
+import GrantsIdentifier from "src/components/GrantsIdentifier";
+import Header from "src/components/Header";
```

Comment:
We had discussed when creating https://github.com/HHS/simpler-grants-gov/pull/2192 that we'd allow imports that are relative if they are in the same directory. We could revisit that, but would be good to update the es lint https://github.com/HHS/simpler-grants-gov/blob/main/frontend/.eslintrc.js#L22 so we are consistent.

Reply:
got it, forgot that we were allowing that. Happy to revert this